examples/asset-explorer/package.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "@blockfrost/asset-explorer",
```

let's keep naming consistent across examples
```suggestion
  "name": "@blockfrost/example-asset-explorer",
```